tools/update-gke-infra.sh

```diff
@@ -155,9 +155,9 @@ function update_to_latest {
 
 function resize_cluster {
     echo "Resizing cluster $CLUSTER_NAME to $CLUSTER_SIZE nodes."
-    read -p 'WARNING: This will delete nodes as well as any associated data on the node. Do you wish to continue? (yes/no) ' ANS
+    read -p 'WARNING: This will delete nodes as well as any associated data on the node. Do you wish to continue? (yes/no) ' AND
```

Comment:
I think this one is intentional (it seems to be short for "answer").

Reply:
@jaegeral ping

Reply:
done